com.ibm.wala.core/src/main/java/com/ibm/wala/ipa/callgraph/impl/Util.java

```diff
@@ -89,7 +89,11 @@ public static void addDefaultSelectors(AnalysisOptions options, IClassHierarchy
    * @throws IllegalArgumentException if cl is null
    * @throws IllegalArgumentException if options is null
    * @throws IllegalArgumentException if scope is null
+   * @deprecated This method is being replaced. Please
+   *     <p>Use{@link Util#addBypassLogic(AnalysisOptions, ClassLoader, String, IClassHierarchy)}
+   *     instead
    */
+  @Deprecated
```

Comment:
In addition to adding this annotation, you should delete the body of the `@Deprecated` method and instead have it just call into the new method.  This way we avoid the duplicated logic